app/views/users/show.html.erb

```diff
@@ -1 +1,38 @@
-A user's details with a list of posts
\ No newline at end of file
+<%= javascript_include_tag "show_more" %>
+<div class="user-wrapper">
+        <div class="user-container">
+            <div class="user-list">
+                <span class="photo">
+                    <%= image_tag(@user.photo, size: "100x100" ) %>
+                </span>
+                <span class="user-info">
+                    <span>
+                        <%= link_to @user.name, user_path(user_id: @user.id, format: :html) %>
+                    </span>
+                    <span>
+                        number of posts: <%= @user.posts_counter ? @user.posts_counter : 0 %>
+                    </span>
+                </span>
+            </div>
+            <div class="bio">
+                <span> Bio </span>
+                <span>
+                    <%=@user.bio%>
+                </span>
+            </div>
+            <ul class="posts">
+                <% @user.posts.each do |post| %>
+                    <li>
+                        <span>
+                            <%= link_to "Post #: #{post.id}" , user_posts_path(user_id: @user.id) %>
+                        </span>
+                        <span>
+                            <%= post.text %>
+                        </span>
+                        <span>comments: <%= post.comments.count %>&nbsp; likes: <%= post.likes.count %></span>
+                    </li>
+                    <% end %>
+            </ul>
+            <button id="show-more-button">Show More</button>
```

Comment:
- [ ] Please as the last reviewer said This button should take you to where you see all posts, currently it doesn't work please address it and answer your question I don't think you need another route for this you can use the index action that shows all the user posts.
![image](https://user-images.githubusercontent.com/74744787/214711317-ad0aa403-179a-4e8e-a349-94009a436d66.png)
![image](https://user-images.githubusercontent.com/74744787/214711719-8bf2a65e-1ccf-4e76-99c1-248dcb3ce42a.png)